app/src/main/java/com/realityexpander/tasky/data/repository/AuthRepositoryImpl.kt

```diff
@@ -14,7 +14,7 @@ class AuthRepositoryImpl(
 ): IAuthRepository {
```

Comment:
It was used quite a bit back in earlier android days, it just makes it super obvious when using an interface versus an Impl. I like it because I dont have to check what it is, i can just look at the name and know its a concrete object or an interface.